svo (setup, misc, empty, funnies, dor).xml

```diff
@@ -1137,11 +1130,9 @@ svo.index_map = pl.tablex.index_map
 -- lostbal_focus, lostbal_herb, lostbal_salve, lostbal_purgative, lostbal_sip
 sk.salvetick, sk.herbtick, sk.focustick, sk.teatick = 0, 0, 0, 0
 sk.purgativetick, sk.siptick, sk.mosstick, sk.dragonhealtick = 0, 0, 0, 0
-sk.smoketick, sk.voicetick, sk.wordtick =  0, 0, 0
+sk.smoketick, sk.voicetick, sk.wordtick, sk.prayertick =  0, 0, 0, 0
+
```

Comment:
You're also creating the prayertick without checking for the Zeal skillset here.